src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -129,14 +130,63 @@ public class MethodologyService : IMethodologyService
                 .OnSuccess(BuildMethodologySummaryViewModel);
         }
 
+        public async Task<Either<ActionResult, List<MethodologyVersionViewModel>>> ListMethodologies(Guid publicationId)
+        {
+            return await _persistenceHelper.CheckEntityExists<Publication>(publicationId,
+                    q => q.Include(p => p.Methodologies)
+                        .ThenInclude(p => p.Methodology)
+                        .ThenInclude(p => p.Versions))
+                .OnSuccess(publication => _userService.CheckCanViewPublication(publication))
+                .OnSuccess(async publication =>
+                {
+                    return await publication.Methodologies
+                        .ToAsyncEnumerable()
+                        .SelectAwait(async publicationMethodology =>
+                        {
+                            var latestVersion = publicationMethodology.Methodology.LatestVersion();
+                            var permissions = new MethodologyVersionPermissions
+                            {
+                                CanDeleteMethodologyVersion =
+                                    await _userService.CheckCanDeleteMethodologyVersion(latestVersion).IsRight(),
+                                CanUpdateMethodologyVersion =
+                                    await _userService.CheckCanUpdateMethodologyVersion(latestVersion).IsRight(),
+                                CanApproveMethodologyVersion =
+                                    await _userService.CheckCanApproveMethodologyVersion(latestVersion).IsRight(),
+                                CanMarkMethodologyVersionAsDraft =
+                                    await _userService.CheckCanMarkMethodologyVersionAsDraft(latestVersion).IsRight(),
+                                CanMakeAmendmentOfMethodology =
+                                    await _userService.CheckCanMakeAmendmentOfMethodology(latestVersion).IsRight(),
+                                CanRemoveMethodologyLink =
+                                    await _userService.CheckCanDropMethodologyLink(publicationMethodology).IsRight()
+                            };
```

Comment:
There's a few reasons why I originally decided not to do this:

* The existing method within `PermissionUtils` which is `GetReleasePermissions` isn't working asynchronously whereas I wanted this to be. So I'd have either had to change that and in turn change things calling that to all work asynchronously, or leave the two methods inconsistent.
* `GetReleasePermissions` only needs the `Release` but a  `GetMethodologyVersionPermissions` will need more than just the `MethodologyVersion`, since it also needs the PublicationMethodology link.
This can be navigated through from the `MethodologyVersion` but we'd probably want to guarantee that it's set before relying on it, or have an extra parameter where the `PublicationMethodology` is passed explicitly. It just means the implementation is not quite as clean when compared to the example of `GetReleasePermissions`.
* Very soon the methodology permissions should only need to be worked out in one place rather than two, and having it inline when building the view model appeared to me to be clearer where they actually come from, rather than needing to click through.

Saying all this though, I've decided to move it here, but not updated `GetReleasePermissions` to work asynchronously just yet.